From b981d900628bb61f498ed7049ae71b8103cd91ea Mon Sep 17 00:00:00 2001
From: Yasufumi Ogawa <ogawa.yasufumi@lab.ntt.co.jp>
Date: Fri, 16 Nov 2018 13:49:25 +0900
Subject: [PATCH 3/5] docs: update how to use section

Update the exmaple of adding vhost because it is changed from `vhost 0`
to specify resource UID as `vhost:0`.

Signed-off-by: Yasufumi Ogawa <ogawa.yasufumi@lab.ntt.co.jp>
---
 docs/guides/setup/howto_use.rst | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/docs/guides/setup/howto_use.rst b/docs/guides/setup/howto_use.rst
index d52a38a..9c7d81d 100644
--- a/docs/guides/setup/howto_use.rst
+++ b/docs/guides/setup/howto_use.rst
@@ -239,11 +239,11 @@ create ``vhost-user`` and ``virtio-net-pci`` devices on the VM.
 
 Before launching VM, you need to prepare a socket file for creating
 ``vhost-user`` device.
-Socket file is created from SPP secondary as following.
+Run ``add`` command with resource UID ``vhost:0`` to create socket file.
 
 .. code-block:: console
 
-    spp > sec 1;add vhost 0
+    spp > sec 1;add vhost:0
 
 In this example, create socket file with index 0 from secondary of ID 1.
 Socket file is created as ``/tmp/sock0``.
-- 
2.13.1

